src/getgb2s.f

```diff
@@ -1,280 +1,248 @@
+C>    This element is actually a pointer to an array
+C>    that holds the data.
+C>    - gfld\%num_coord number of values in array gfld\%coord_list.
+C>    - gfld\%ndpts Number of data points unpacked and returned.
+C>    Note that this number may be different from the value of
+C>    - gfld\%ngrdpts if a logical bitmap is in use with grid points
+C>    that are being masked out.
+C>    - gfld\%idrtnum Data Representation Template Number (Code Table 5.0)
+C>    - gfld\%idrtmpl Contains the data values for the specified Data
+C>    Representation Template (N=gfld\%idrtnum). Each element of this
+C>    integer array contains an entry (in the order specified) of
+C>    Product Defintion Template 5.N. This element is actually a
+C>    pointer to an array that holds the data.
+C>    - gfld\%idrtlen Number of elements in gfld\%idrtmpl. i.e. number
+C>    of entries in Data Representation Template 5.N (N=gfld\%idrtnum).
+C>    - gfld\%unpacked logical value indicating whether the bitmap and
+C>    data values were unpacked. If false, gfld\%bmap and gfld\%fld
+C>    pointers are nullified.
+C>    - gfld\%expanded Logical value indicating whether the data field
+C>    was expanded to the grid in the case where a bit-map is present.
+C>    If true, the data points in gfld\%fld match the grid points and
+C>    zeros were inserted at grid points where data was bit-mapped out.
+C>    If false, the data values in gfld\%fld were not expanded to the
+C>    grid and are just a consecutive array of data points corresponding
+C>    to each value of "1" in gfld\%bmap.
+C>    - gfld\%ibmap Bitmap indicator (see Code Table 6.0)
+C>     - 0 bitmap applies and is included in Section 6.
+C>     - 1-253 Predefined bitmap applies
+C>     - 254 Previously defined bitmap applies to this field
+C>     - 255 Bit map does not apply to this product.
+C>    - gfld\%bmap Logical*1 array containing decoded bitmap, if ibmap=0
+C>    or ibap=254. Otherwise nullified. This element is actually a
+C>    pointer to an array that holds the data.
+C>    - gfld\%fld Array of gfld\%ndpts unpacked data points. This element
+C>    is actually a pointer to an array that holds the data.
+C>    @param[out] LPOS starting position of the found index record
+C>    within the complete index buffer, CBUF. = 0, if request not found.
+C>    @param[out] IRET integer return code
+C>    - 0 all ok
+C>    - 97 error reading grib file
+C>    - other gf_getfld grib2 unpacker return code
+C>
+C>    @note This subprogram is intended for private use by getgb2
+C>    routines only. Note that derived type gribfield contains pointers
+C>    to many arrays of data. The memory for these arrays is allocated
+C>    when the values in the arrays are set, to help minimize problems
+C>    with array overloading. Because of this users are encouraged to
+C>    free up this memory, when it is no longer needed, by an explicit
+C>    call to subroutine gf_free().
+C>
+C>    @author Stephen Gilbert @date 2002-01-15
```

Comment:
@Hang-Lei-NOAA @edwardhartnett Do we need two @author lines? there's one on line 4 already.

Reply:
One for the file, one for each subprogram in the file.

Reply:
Thanks